src/Stubs/common/FileLocator.stubphp

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Symfony\Component\Config;
+
+class FileLocator implements FileLocatorInterface
```

Comment:
I think we should create a stub for the `FileLocatorInterface` - as it's explicitly part of the contract of the interface:

```
    /**
     * Returns a full path for a given file name.
     *
     * @param string      $name        The file name to locate
     * @param string|null $currentPath The current path
     * @param bool        $first       Whether to return the first occurrence or an array of filenames
     *
     * @return string|array The full path to the file or an array of file paths
     *      
     * @throws \InvalidArgumentException        If $name is empty
     * @throws FileLocatorFileNotFoundException If a file is not found
     */
    public function locate(string $name, string $currentPath = null, bool $first = true);
```

Reply:
Good point! Updated